sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -74,14 +73,8 @@
 
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-identity-providers-jdbc-mysql</artifactId>
-        <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-identity-providers-jdbc-mysql;dependency} -->
-      </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-identity-providers-jdbc-postgresql</artifactId>
-        <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-identity-providers-jdbc-postgresql;dependency} -->
+        <artifactId>azure-identity-extensions</artifactId>
+        <version>1.0.0</version> <!-- {x-version-update;unreleased_com.azure:azure-identity-extensions;dependency} -->
```

Comment:
since azure-identity-extensions has been released, can we use the GA version here?

Reply:
still will use another PR.